src/dodal/devices/eiger.py

```diff
@@ -144,6 +144,9 @@ def stop(self, *args):
         self.disarm_detector()
         stop_status &= self.disable_roi_mode()
         stop_status.wait(self.GENERAL_STATUS_TIMEOUT)
+        # See https://github.com/DiamondLightSource/hyperion/issues/1395
+        LOGGER.info("Turning off Eiger dev/shm streaming")
```

Comment:
Nit: If we have `enable_dev_shm` set to off, I think we will get a logging message during `change_dev_shm`, and in Hyperion when XRC finishes, and again here - all saying that dev shm is being disabled

Reply:
I think that given the consequence of leaving it on accidentally is very bad it's actually good to be very specific in the logs that we're turning it off even if that doesn't actually matter

Reply:
yeah, makes sense